src/bundle/Resources/views/themes/admin/limitation/base_limitation_values.html.twig

```diff
@@ -8,8 +8,13 @@
             }
         }) }}
 
+        {% set choices = [] %}
+        {% for choice in form.limitationValues.vars.choices %}
+            {% set choices = choices|merge(choice.choices is defined ? choice.choices : [choice]) %}
+        {% endfor %}
```

Comment:
Maybe something more explicit? We might also want to wrap this with `{% with %}` tag to prevent overwriting `choices` variable, in case it's already defined in template.

```suggestion
        {% set choices = form.limitationValues.vars.choices | reduce((carry, choice) => carry|merge(choice.choices is defined ? choice.choices : [choice]), []) %}
```

Reply:
Or even:

```suggestion
        {% set choices = form.limitationValues.vars.choices
            | map(choice => choice.choices is defined ? choice.choices : [choice])
            | reduce((carry, choices) => carry|merge(choices), []) 
        %}
```